pkg/registries/registries.go

```diff
@@ -229,8 +231,7 @@ func (ms *mirrorSet) generateConfig(secretsConfig credentialprovider.DockerConfi
 		result += hostCapabilities
 		result += "\r\n"
 
-		// TODO: Remove this when we support TLS auth using CA certs - https://issues.redhat.com/browse/WINC-1291
-		result += "  skip_verify = true"
```

Comment:
What is the default value for `skip_verify`?  should it be explicitly set to not `true`

Reply:
Default value is false https://github.com/containerd/containerd/blob/main/docs/hosts.md

Reply:
ACK, thanks for checking this out.